### PR TITLE
Fix Wallaby test script in CI

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes
**Asana Ticket:** [Wallaby tests don't pass in CI](https://app.asana.com/0/385363666817452/1199728396564607/f)

The script we ran always passed, even when the tests inside fail.

This update fixes the script (thanks @paulswartz) and also updates a few integration tests to bring them to passing. The old tests were mainly outdated because they were using CSS selectors that became invalid after we refactored the customer support form some months ago.

### CODE REVIEW

**Review by alicia-mbta — APPROVED** [review]
Looks good to me! Thanks for the added tests 🙏

File: apps/site/test/site_web/integration/support_form_test.exs [hunk 1]
```diff
@@ -3,117 +3,114 @@ defmodule CustomerSupportTest do
...
   describe "customer support form" do
     @tag :wallaby
     test "Shows an error for missing type and comments", %{session: session} do
       session
-      |> visit("/customer-support")
-      |> assert_has(@submit_button)
-      |> click(@submit_button)
-      |> assert_has(css(".form-group.has-danger #comments"))
-      |> assert_has(css(".form-group.has-danger #service"))
+      |> Browser.visit("/customer-support")
```
Comment (alicia-mbta) [inline apps/site/test/site_web/integration/support_form_test.exs:28]: I'm not very familiar with Wallaby, what is the difference between calling `visit` and `Browser.visit`?

**Review by alicia-mbta — APPROVED** [review]
Looks good. Also thanks for the additional tests.
I will try to incorporate Wallaby from now on because I don't think I was really using it.

**Comment by thecristen** [comment]
Addressed in #1021

**Pull request closed without merging**